src/frontend/components/ViewersList/components/ViewersListItem/index.tsx

```diff
@@ -9,15 +9,15 @@ const StyledText = styled(Text)`
   letter-spacing: 0.07px;
 `;
 
-interface StudentViewersListItemProps {
+interface ViewersListItemProps {
   isInstructor: boolean;
   name: string;
 }
 
-export const StudentViewersListItem = ({
+export const ViewersListItem = ({
```

Comment:
for some reasons it seems like we lost the round effect here aswell in the snapshot

Reply:
address above in this PR: 

> To create the border all around the avatar (a border only shown when the avatar is the one of an instructor), I was using the border props of grommet. But in this PR I stop to use this border props and use the outline css property instead. (I decided to switch border and outline because outline doesn't take any space, so it's more confortable to use in this precise case).